doc/modulefile_templates/openmpi

```diff
@@ -31,7 +31,7 @@ unsetenv SLURM_EXPORT_ENV
 setenv PSM2_PATH_SELECTION "static_base"
 setenv SLURM_CPU_BIND "none"
 
-# Settings specific for Cheyenne
+# Settings specific for Cheyenne and Hercules
```

Comment:
just wanted to check: why are these settings only needed for cheyenne and hercules?

Reply:
Because every HPC has its own settings, depending on the interconnect, the scheduler settings etc.